### DISCUSSION TITLE
Install plugins directly from the github repository

### DISCUSSION THREAD

**Discussion by acseven** [Budibase/budibase#8541] [Ideas]
Right now to install a plugin it is necessary to specify a URL, including those listed on the project's plugin listing (https://github.com/Budibase/plugins). I suggest plugins listed there to be indexed in a structured repository data source that could be used by the core budibase installation, allowing direct installation / upgrade of plugins based on that data source.

This sounds a bit obvious, but I tried to find a similar suggestion in other discussions but didn't find any.

**Comment by melohagan** [comment]
Hey @acseven 

So if I understand correctly, you would like to provide a link to a folder what would import all plugins found inside the folder?

**Reply by acseven** [reply]
Hi, no not really. 
Just to have the plugins listed on the budibase/plugins repository appear in a list on our own budibase installation, and from there have buttons to install/remove/upgrade.

**Reply by melohagan** [reply]
Ah I got you now. That would be pretty cool! I think in the future at some point the idea would be to have some sort of plugins marketplace.

**Reply by acseven** [reply]
Yeah, that makes sense!
Thanks!
